src/components/flat-table/flat-table-row/flat-table-row.component.js

```diff
@@ -31,6 +31,7 @@ const FlatTableRow = React.forwardRef(
     ref
   ) => {
     const [isExpanded, setIsExpanded] = useState(expanded);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I've pulled the branch down locally and this doesn't flag, you should remove this